packages/amplify-graphql-schema-generator/src/ts-schema-generator/helpers.ts

```diff
@@ -0,0 +1,157 @@
+import ts from 'typescript';
+import { Field, FieldType, Model, Schema } from '../schema-representation';
+
+const SCHEMA_PACKAGE = '@aws-amplify/data-schema';
+const MODEL_METHOD = 'model';
+const SCHEMA_METHOD = 'schema';
+const IDENTIFIER_METHOD = 'identifier';
+const ARRAY_METHOD = 'array';
+const REQUIRED_METHOD = 'required';
+const STRING_METHOD = 'string';
+const ENUM_METHOD = 'enum';
+const REFERENCE_A = 'a';
+const EXPORT_VARIABLE_NAME = 'schema';
```

Comment:
nit: would be nice to encapsulate the known gen2 code-first methods into an interface or structure rather than free-floating constants